qa/tasks/cephfs/test_failover.py

```diff
@@ -340,6 +340,23 @@ def test_thrash(self):
 
         self.fs.wait_for_daemons(timeout=90)
 
+class TestFailoverBeaconHealth(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 1
+
+    def test_replay_beacon_estimated_time(self):
+        """
+        That beacon emits warning message with estimated time to complete replay
+        """
+        self.config_set("mds", "delay_journal_replay_for_testing", "true")
+        self.config_set("mds", "delay_time_journal_replay_for_testing", "5")
+        original_active = self.fs.get_active_names()[0]
+        self.mounts[0].test_files = [str(x) for x in range(100)]
+        self.mounts[0].create_files()
+        self.fs.mds_restart(original_active)
+        self.wait_for_health("MDS_ESTIMATED_REPLAY_TIME", 60)
```

Comment:
Add a test to verify the estimate is accurate.

Reply:
```self.mounts[0].test_files = [str(x) for x in range(100)]
        self.mounts[0].create_files()```
This creates 100 files.

Reply:
Ah, yes that's true.